pipes/using_pipes.md

```diff
@@ -1,20 +1,22 @@
 # Using Pipes
 
-Like a filter, a pipe also takes data as input and transforms it to the desired output. A basic example of using pipes is shown below:
+Like a filter, a pipe also takes data as input and transforms it into the desired output. To use a pipe, you can use the pipe operator ( | ) followed by the name of the pipe function inside the interpolation expression. All pipes work this way. A basic example of using pipes is shown below:
 
 ```javascript
 import { Component } from '@angular/core';
 
 @Component({
   selector: 'product-price',
-  template: `<p>Total price of product is {{ price | currency }}</p>`
+  template: `<p>Total price of product is {{ price | currency }}</p>`,
```

Comment:
The trailing comma as it might be confusing for beginners, should we leave out ?